quesma/model/metrics_aggregations/top_hits.go

```diff
@@ -50,49 +46,11 @@ func (query TopHits) TranslateSqlResponseToJson(rows []model.QueryResultRow) mod
 		sourceMap := model.JsonMap{}
 
 		for _, col := range valuesForHits {
-			var withoutQuotes string
```

Comment:
Just to clarify, it this functionality moved to transformation phase? or it's not needed anymore

Reply:
Yes. It's handled generically.